src/taxi/utils.py

```diff
@@ -0,0 +1,127 @@
+import os
+from enum import Enum
+from django.conf import settings
+from odata_request_parser.main import OdataFilterParser, OdataSelectParser
+
+from main.utils import DecosBase, ImmediateHttpResponse
+
+
+class DecosBusiness(Enum):
+    taxi = "TAXXXI"
+    folders = "FOLDERS"
```

Comment:
Deze vars lijken nogal redundant (folders is folders). Ik zou er een naam aangeven wat het is. Dus ipv "taxi" iets als "username" (dat is het toch?).